packages/jest-worker/src/__tests__/Queue.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
```

Comment:
no need for `user strict` when using `import` - it's implicit

Reply:
I cloned this template from `Farm.test.js` but I will remove this line